### PR TITLE
Show GitHub and npm urls

### PR DESCRIPTION
**Pull request by nidkil**
This is related to [issue #417](https://github.com/webpro/release-it/issues/417).

Added message that is displayed on completion with URLs to go to the GitHub repository and the NPM package. In a lot of terminals this URLs are displayed as clickable making it easy to jump to them and do a quick check.

### CODE REVIEW

**Comment by nidkil** [comment]
I forgot to merge the upstream before making the changes. Fixed the code after merging upstream.

**Comment by nidkil** [comment]
Fixed failing test.

File: lib/config.js [hunk 1]
```diff
@@ -127,7 +127,8 @@ class Config {
       version,
       name,
       private: isPrivate,
-      publish: !!name
+      publish: !!name,
+      packageUrl: 'https://www.npmjs.com/package/' + name
```
Comment (webpro) [inline lib/config.js]: Please add a helper to the `/lib/npm` module to provide this url.

File: lib/tasks.js [hunk 2]
```diff
@@ -271,6 +271,19 @@ module.exports = async options => {
       await run(`!rm -rf ${dist.stageDir}`);
     }
 
+    if (github.release) {
+      getSpinner().stopAndPersist({
```
Comment (webpro) [inline lib/tasks.js]: My bad, but please use the `log` helper for this.

File: lib/tasks.js [hunk 3]
```diff
@@ -271,6 +271,19 @@ module.exports = async options => {
...
+        symbol: `❗`,
+        text: `To Github repository ${config.options.repo.remote}`
```
Comment (webpro) [inline lib/tasks.js]: * I'd prefer a simple `🔗 [url]` here. I think it's clear from the url alone what we're linking to.
* The url should be coming from the return value of `release`: `releaseInfo.html_url` (not the repo url, which could be a non-web url, but the newly created release page).

File: lib/tasks.js [hunk 4]
```diff
@@ -271,6 +271,19 @@ module.exports = async options => {
...
+      });
+    }
+    if (npm.publish) {
+      getSpinner().stopAndPersist({
+        symbol: `❗`,
+        text: `To NPM package ${config.options.npm.packageUrl}`
+      });
```
Comment (webpro) [inline lib/tasks.js]: Something like this should be enough here:

```
log(`🔗 ${npm.getPackageUrl()}`);
```

File: lib/config.js (hunk 1 above)
Comment (nidkil) [reply lib/config.js]: Will do.

File: lib/tasks.js (hunk 2 above)
Comment (nidkil) [reply lib/tasks.js]: Okay.

File: lib/tasks.js (hunk 4 above)
Comment (nidkil) [reply lib/tasks.js]: Okay.

**Comment by webpro** [comment]
Thanks @nidkil! The new feature is released in [v9.2.0](https://github.com/webpro/release-it/releases/tag/9.2.0).

**Pull request merged**